optimum/intel/openvino/modeling_visual_language.py

```diff
@@ -285,7 +296,7 @@ def _save_pretrained(self, save_directory: Union[str, Path]):
         dst_file_names = [
             "openvino_language_model.xml",
             "openvino_text_embeddings_model.xml",
-            "openvino_vision_embeddings.xml",
+            "openvino_vision_embeddings_model.xml",
```

Comment:
could be all added to https://github.com/huggingface/optimum-intel/blob/cead516b3b8c38c1a0b8b8937a861edccdfab839/optimum/intel/openvino/utils.py#L53 to avoid potential issues

Reply:
moved